src/morphosource/exceptions.py

```diff
@@ -4,3 +4,7 @@ class RestrictedDownloadError(Exception):
 
 class ItemNotFound(Exception):
     pass
+
+
+class MetadataMissingError(Exception):
+    pass
```

Comment:
```suggestion
    pass
    
```

Reply:
It seems to have still added a space to the last line or something, GH is reading it as not having a blank last line.